webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -386,91 +386,91 @@ class Boot extends Loggable {
             >> LocGroup(name+"Group")
             >> TestAccess( () => userIsAllowed("/secure/index",AuthorizationType.Parameter.Read ) )
       )
     }
 
     def administrationMenu =
-      (Menu("AdministrationHome", <i class="fa fa-gear"></i> ++ <span>Administration</span>: NodeSeq) /
+      (Menu("40", <i class="fa fa-gear"></i> ++ <span>Administration</span>: NodeSeq) /
         "secure" / "administration" / "index" >> TestAccess ( ()
             => userIsAllowed("/secure/index",AuthorizationType.Administration.Read, AuthorizationType.Technique.Read) )).submenus (
 
-          Menu("Setup", <span>Setup</span>) /
+          Menu("10", <span>Setup</span>) /
             "secure" / "administration" / "setup"
             >> LocGroup("administrationGroup")
             >> TestAccess( () => userIsAllowed("/secure/index",AuthorizationType.Administration.Read ) )
 
-        , Menu("policyServerManagement", <span>Settings</span>) /
+        , Menu("20", <span>Settings</span>) /
             "secure" / "administration" / "policyServerManagement"
             >> LocGroup("administrationGroup")
             >> TestAccess ( () => userIsAllowed("/secure/index",AuthorizationType.Administration.Read) )
 
-        , Menu("databaseManagement", <span>Reports database</span>) /
+        , Menu("30", <span>Reports database</span>) /
             "secure" / "administration" / "databaseManagement"
             >> LocGroup("administrationGroup")
             >> TestAccess ( () => userIsAllowed("/secure/administration/policyServerManagement",AuthorizationType.Administration.Read) )
 
-        , Menu("TechniqueLibraryManagement", <span>Techniques tree</span>) /
+        , Menu("40", <span>Techniques tree</span>) /
             "secure" / "administration" / "techniqueLibraryManagement"
             >> LocGroup("administrationGroup")
             >> TestAccess( () => userIsAllowed("/secure/index",AuthorizationType.Technique.Read ) )
 
-        , Menu("apiManagement", <span>API accounts</span>) /
+        , Menu("50", <span>API accounts</span>) /
             "secure" / "administration" / "apiManagement"
             >> LocGroup("administrationGroup")
             >> TestAccess ( () => userIsAllowed("/secure/administration/policyServerManagement",AuthorizationType.Administration.Write) )
       )
 
     def pluginsMenu = {
-      (Menu("PluginsHome", <i class="fa fa-puzzle-piece"></i> ++ <span>Plugins</span> ++ <span data-lift="PluginExpirationInfo.renderIcon"></span>: NodeSeq) /
+      (Menu("50 - plugins", <i class="fa fa-puzzle-piece"></i> ++ <span>Plugins</span> ++ <span data-lift="PluginExpirationInfo.renderIcon"></span>: NodeSeq) /
         "secure" / "plugins" / "index"
         >> LocGroup("pluginsGroup")
         >> TestAccess ( () => userIsAllowed("/secure/index", AuthorizationType.Administration.Read)
       )) submenus (
-          Menu("pluginInformation", <span>Plugin information</span>) /
+          Menu("0", <span>Plugin information</span>) /
```

Comment:
I think I would use `01` here to let the possibility to have things before in the future, in the middle of a release cycle.